pkg/synthetictests/duplicated_events.go

```diff
@@ -539,6 +540,33 @@ func isConsoleReadinessDuringInstallation(monitorEvent monitorapi.EventInterval,
 	return isEventDuringInstallation(monitorEvent, kubeClientConfig, regExp)
 }
 
+// isConfigOperatorReadinessFailed returns true if the event matches a readinessFailed error that timed out
+// in the openshift-config-operator.
+// like this:
+// ...ReadinessFailed Get \"https://10.130.0.16:8443/healthz\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"
+func isConfigOperatorReadinessFailed(monitorEvent monitorapi.EventInterval, _ *rest.Config, _ int) (bool, error) {
+	regExp := regexp.MustCompile(probeTimeoutMessageRegExpStr)
+	return isConfigOperatorReadinessrobeFailedMessage(monitorEvent, regExp), nil
+}
+
+func isConfigOperatorReadinessrobeFailedMessage(monitorEvent monitorapi.EventInterval, regExp *regexp.Regexp) bool {
```

Comment:
robe typo

Reply:
I can't believe I missed that :-) thanks for catching it.